### ISSUE TITLE
dotnet dev-certs does not "trust" the generated certificate

### ISSUE THREAD

**Issue by nathanjwtx** [dotnet/aspnetcore#35354]
### Describe the bug
When I try to run `dotnet dev-certs https --trust` the generated certificate is not added to the Trusted certificates folder, just to the Personal certificates folder. I've tried `--clean` and then `--trust` again but with the same result. I have tried with different versions of the SDK installed but no difference in behaviour. By this I mean I have installed each version individually and tried the certificate command.

One suggestion I have tried is to "copy" the certificate from Personal to Trusted but this doesn't work either. The cert is copied but disappears on refresh.

I have also clicked on the warning dialogue that pops up in Rider and VS to say a dev cert needs to be created but this still does not resolve to a trusted certificate.

### To Reproduce

I am not totally sure how to reproduce this problem. It seems to either works on your machine or it doesn't. I am also unsure if this specifically related to ASP.Net or is something specific to my machine and Windows build.

I know there is no project attached but am hoping you might have some suggestions I could try in order to resolve this. Or at least confirm if it is or is not an ASP.Net specific problem.

### Exceptions (if any)

```
System.InvalidOperationException: IDX20803: Unable to obtain configuration from: 'System.String'.
 ---> System.IO.IOException: IDX20804: Unable to retrieve document from: 'System.String'.
 ---> System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: UntrustedRoot
```

### Further technical details
- ASP.NET Core version 3, 5, 6 (see below) 
- Include the output of `dotnet --info`
```
.NET SDK (reflecting any global.json):
 Version:   6.0.100-preview.7.21379.14
 Commit:    22d70b47bc

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19043
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.100-preview.7.21379.14\

Host (useful for support):
  Version: 6.0.0-preview.7.21377.19
  Commit:  91ba01788d

.NET SDKs installed:
  3.1.412 [C:\Program Files\dotnet\sdk]
  5.0.302 [C:\Program Files\dotnet\sdk]
  6.0.100-preview.7.21379.14 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.8 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.0-preview.7.21378.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.8 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0-preview.7.21377.19 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.0-preview.7.21378.9 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version
Rider 2021, VS 2022 Preview, VS 2019 Pro, VS Code Insiders

**Comment by nathanjwtx** [comment]
I finally stumbled upon the answer; my anti-virus, WebRoot. I was following a YouTube tutorial on how to add a custom certificate to Kestrel and in doing so I discovered that WebRoot was blocking access to the `hosts` file. Disabling the av allowed me to update that file but also, it then allowed trusting of the dev-cert generated by `dotnet dev-certs https --trust`. Not sure how to prevent that in future other than to remember to temporarily disable the av when generating the certificates.
Thank you for your time.
